ambry-utils/src/test/java/com.github.ambry.utils/TestUtils.java

```diff
@@ -223,6 +229,62 @@ public static String getRandomKey(int size) {
     return sb.toString();
   }
 
+  /**
+   * A wrapper class to start and shutdown {@link ZooKeeperServer}.
+   * We maintain this class because {@link org.I0Itec.zkclient.ZkServer} calls NetworkUtil.getLocalHostNames()
+   * which takes time in Mac OS.
```

Comment:
and that it is unnecessary for our tests?

Reply:
ZKServer calls NetworkUtil.getLocalHostNames() to
1. logging
2. make sure `"localhost"` is in the list of `NetworkUtil.getLocalHostNames()`. If not, it does nothing(doesn't start server).

In test, I think we are good to go without above.

Reply:
makes sense - might as well add this whole reasoning in the comment.